test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  it ("returns a reversed array of the number's digits", () => {
+    expect (numberToReversedDigits([12345])).toStrictEqual([5, 4, 3, 2, 1])
```

Comment:
It would be good to add a few more test cases here to make sure the code doesnt only work for this one example, maybe some of different lengths would be a good place to start?